src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/AttributeStringLiteralsShouldParseCorrectly.cs

```diff
@@ -243,20 +243,18 @@ private static void Analyze(Action<Diagnostic> reportDiagnostic, AttributeData a
 
     internal class ValueValidator
     {
-        private readonly string? _ignoredName;
+        private readonly string[]? _ignoredNames;
```

Comment:
The compiler doesn't pass a null array for params.

You can just use `string[]`.

Reply:
+1 You can also get rid of null check in `IsIgnoredName`.